Formula/a/autobrr.rb

```diff
@@ -0,0 +1,45 @@
+class Autobrr < Formula
+  desc "Modern, easy to use download automation for torrents and usenet"
+  homepage "https://autobrr.com/"
+  url "https://github.com/autobrr/autobrr/archive/refs/tags/v1.36.0.tar.gz"
+  sha256 "631ee5f858c5952e1d7a8c27b13fe33aea9021ffd814bb540f0e494291168309"
+  license "GPL-2.0-or-later"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    # build the frontend
+    system "npx", "pnpm", "install", "--dir", "web"
+    system "npx", "pnpm", "--dir", "web", "run", "build"
+
+    # compile flags to embed version information
+    ldflags = "-s -w -X main.version=#{version} -X main.commit=#{tap.user}"
+
+    system "go", "build", *std_go_args(output: bin/"autobrr", ldflags: ldflags), "./cmd/autobrr"
+    system "go", "build", *std_go_args(output: bin/"autobrrctl", ldflags: ldflags), "./cmd/autobrrctl"
+  end
+
+  def post_install
+    (var/"autobrr").mkpath
+  end
+
+  service do
+    run [opt_bin/"autobrr", "--config", var/"autobrr/config.toml"]
+    keep_alive true
+    log_path var/"log/autobrr.log"
+  end
+
+  def caveats
+    <<~EOS
+      autobrr will generate a default configuration file at first run located at:
+        #{var}/autobrr/config.toml
+
+      Please modify this file as necessary after running autobrr for the first time.
```

Comment:
Is this something specific to packaging `autobrr` in Homebrew? Or is this the default behavior? If it's the latter, this should be removed since caveats are reserved for idiosyncrasies with Homebrew packaging only.

Reply:
This is indeed the default behavior 😃 Removed now 👍